### DISCUSSION TITLE
Change to shipping

### DISCUSSION THREAD

**Discussion by lscheffler** [fdbozzo/foxbin2prg#46] [Ideas]
The repo ships both the PJX table and the EXE.   
Since the prg is part of this, it would be enough to ship the pjm and instruction to create the pjx (and compile).    
While working with the code, I found it not nice that _git_ was complaining about this table all the time.

What is your view about this?

For the Thor folks, could Thor create the pjx and compile to EXE on update?

Update:
Somehow it needs the EXE to run. Run as prg, it gathers at least the Version number from the exe.  So this is only about the PJX .

**Comment by DougHennig** [comment]
Normally, I would think the PJX, PJT, and EXE files should be excluded from the repository. We could add instructions to README that tells the user how to create the PJX/PJT files by running the PRG and then create the EXE.

However, I think this will cause problems for users -- they will not read the documentation and then they will not be able to use FoxBin2PRG and there will be lots of issues created here.

So, we should leave them in the repository even though it causes problems for us.

**Comment by lscheffler** [comment]
This looks like it is working like it is.
